content/NavCoin Core/How to move from NavPi to Raspberry Pi 4 and CM4.en.md

```diff
@@ -0,0 +1,182 @@
+
+**Disclaimer**: This how to was made to work with a lot of googeling and asking a ton of stupind questions to the patient Discord members of the Navcoin #navpi-support channel. It may not be the 'only' or 'best' way BUT this is how I was able to get staking working for me.
+
+**Special Thanks to `@salmonskinroll` and `@aguycalled` as well as `u/Zane_TLI` on Reddit for this [post](https://www.reddit.com/r/NavCoin/comments/l8jdcr/staking_with_navcoind_getstakinginfo_return>)**
+
+This how to is using the CM4 [CM4104032](https://www.buyapi.ca/product/raspberry-pi-compute-module-4-wireless-4gb-32gb-cm4104032/).
+
+1. Assumption is you have a running CM4 (how to can be found [here](https://www.youtube.com/watch?reload=9&v=jp_mF1RknU4)) and ssh access to it
+
+
+   **Notes**:
+
+- I had to use Raspbian 32bit as I believe the Navcoin binaries only come in 32-bit for arm (if I learn otherwise I will update)
+- I also am utilizing the `screen` window session manager to keep things running in case you get disconnected. [Here is a quick intro on how to use `screen`](https://www.youtube.com/watch?v=hB6Y72DK8mc)
+
+```
+pi@raspberrypi:~ $ screen
+```
+
+**Note**: When reconnecting to the Raspberry Pi remotely re-connect to the still running window session(s)
+
+```
+pi@raspberrypi:~ $ screen -x
+```
+
+          - I am performing all commands from the /home/pi directory
+          - At time of writing the latest version of navcoin was 6.1.0
+
+2. Get the latest Navcoin files from [here](https://build.nav.community/binaries/master/navcoin-6.1.0-arm-linux-gnueabihf.tar.gz)
+
+```
+pi@raspberrypi:~ $ wget https://build.nav.community/binaries/master/navcoin-6.1.0-arm-linux-gnueabihf.tar.gz
+```
+
+3. Once downloaded untar the tarball file 
+
+```
+pi@raspberrypi:~ $ tar -zxvf navcoin-6.1.0-arm-linux-gnueabihf.tar.gz
+```
+
+4. Bootstrap you wallet
+
+##### What is “Bootstrapping”? 
+
+Bootstrapping is the process of adding a pre-synced blockchain from a trusted source to your wallet, allowing you to skip part or all of the blockchain syncing process. see [this article](/navcoin-core/bootstrap-your-wallet/)
+
+First let's create the navcoin data directory. For more info please see [this article](/navcoin-core/locate-your-data-directory/)
+```
+pi@raspberrypi:~ $ mkdir .navcoin4
+```
+Now we download the pre-synched bloackchain inside the .navcoin4 data directory
+```
+pi@raspberrypi:~ $ cd .navcoin4
+pi@raspberrypi:~/.navcoin4 $ wget https://bootstrap.nav.community/bootstrap.tar.gz
+```
+
+**Note**: this may take some time as this file is around 3.3GB in size
+
+Then we untar the blockchain tarball
+
+```
+pi@raspberrypi:~/.navcoin4 $  tar -zxvf bootstrap.tar.gz
+```
+
+5. Copy your `wallet.dat` file from the NavPi data directry `~/.navcoin4/` to the new Paspberry Pi data directry `~/.navcoin4/`
```

Comment:
up to this point, all previous steps are the same for anyone who tries to set up a headless node. So it may be a good idea to have a note saying these steps are only required if moving from navpi to make the whole article more universal

Reply:
Agreed - will update